src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -131,7 +131,13 @@ final class ExposureNotificationSettingViewController: UITableViewController, Ac
 		let content = model.content[section]
 
 		guard content.cellType == .euTracingCell else { return }
-
+		
+		if #available(iOS 13, *) {
+			navigationItem.largeTitleDisplayMode = .always
+		} else {
+			navigationItem.largeTitleDisplayMode = .never
+		}
+		navigationItem.largeTitleDisplayMode = .never
```

Comment:
Shouldn't this line be gone? Right now it's always set to .never I think.

Reply:
yes you are right